crates/bevy_reflect/bevy_reflect_derive/src/container_attributes.rs

```diff
@@ -172,12 +171,12 @@ impl ReflectTraits {
                 // Handles `#[reflect( Hash(custom_hash_fn) )]`
                 NestedMeta::Meta(Meta::List(list)) => {
                     // Get the first ident in the path (hopefully the path only contains one and not `std::hash::Hash`)
-                    let ident = if let Some(segment) = list.path.segments.iter().next() {
-                        segment.ident.to_string()
-                    } else {
+                    let Some(segment) = list.path.segments.iter().next() else {
                         continue;
                     };
 
+                    let ident = segment.ident.to_string();
+
                     // Track the span where the trait is implemented for future errors
                     let span = ident.span();
```

Comment:
Is this intended? In the code above the `.span()` is called on `segment.ident`, here it's called on `segment.ident.to_string()`.

I would also suggest to rename `ident` to `ident_name` here, to stay consistent with the code above.

Reply:
That's what the code did before too.

Reply:
Yeah this was a minimal refactoring / thats what the name was before. But I'm happy to make targeted improvements and I agree this name could be better.